### ISSUE TITLE
Explorer says "We encountered an error. Please try again later."

### ISSUE THREAD

**Issue by metacoin** [Blockstream/esplora#28]
Environment:
Ubuntu 18.04.1 LTS

Steps to reproduce:
Followed README...
```
$ git clone https://github.com/Blockstream/esplora && cd esplora
$ npm install
$ export API_URL=http://localhost:3000/ # or https://blockstream.info/api/ if you don't have a local API server
# (see more config options below)
$ npm start
```

Also running from the docker command at the same time:

```
docker run -p 8080:80 \
           --volume $PWD/data_bitcoin_mainnet:/data \
           --rm -i -t esplora \
           bash -c "/srv/explorer/run.sh bitcoin-mainnet explorer"
```

The line `export API_URL=http://localhost:3000/` is confusing because I'm not sure what is supposed to be running on port 3000. `netstat -plunt` only shows SSH, the explorer front-end running on port 5000, and the explorer API(?) running on port 8080. 

```
Proto Recv-Q Send-Q Local Address           Foreign Address         State       PID/Program name    
tcp6       0      0 :::22                   :::*                    LISTEN      -                   
tcp6       0      0 :::5000                 :::*                    LISTEN      28567/node          
tcp6       0      0 :::8080                 :::*                    LISTEN      - 
```

Even with `export API_URL=http://localhost:8080/` I am getting the same results. 

What is supposed to be running on port 3000?

Attached is a screenshot of the error page, which is what I see when I navigate to port 5000,
![screenshot from 2018-12-29 13-04-03](https://user-images.githubusercontent.com/4959276/50540979-44c96900-0b6a-11e9-8b51-e639e86bbade.png)

**Comment by greenaddress** [comment]
@metacoin thank you for the report

The first step you quote from the README indeed implies that you are already running on localhost an instance of blockstream/electrs (a modified version of the original electrs) and you can use blockstream instance if you just want to test out/setup your local config (as per shell comment).

As per the docker run, it will take several hours if not days depending on how powerful is the machine to synchronize. We should improve the README and clarify the above points as well as improve the explorer to inform the status of the synchronization directly from the explorer on a web page and end point.

**Comment by greenaddress** [comment]
@metacoin see https://github.com/Blockstream/esplora/issues/26#issuecomment-450512534 (you can either read them directly, which may need sudo because docker sets the file as readable by root only) or you can enter the docker instance with 'exec' and tail from within.

**Comment by metacoin** [comment]
Thanks, I was looking for a status update on which block the explorer was up-to-date with, though looking through the logs I don't think it's possible to evaluate that at this time. 

Also, I am still confused about what might be running on port 3000, as both `netstat` and `ss` report nothing running on port 3000. I'm connecting to port 8080 to view the explorer front-end now and it seems up to date with other block explorers on the web. Maybe I have a misunderstanding about how that piece of the software works.

As I posted in my other comment, I fixed this problem on my end, after hunting through logs the issue was not enough HDD space on my device.

https://github.com/Blockstream/esplora/issues/26#issuecomment-450666694
